common/mam/v1/BUILD

```diff
@@ -0,0 +1,32 @@
+package(default_visibility=["//visibility:public"])
+
+cc_library(name="mam",
+           srcs=["mam.c"],
+           hdrs=["mam.h"],
+           include_prefix="mam",
```

Comment:
let's not specify a specific include prefix here so that people have to use the versioned path by default